docs/docker_usage.md

```diff
@@ -61,3 +74,35 @@ servicedirectory \
 --adaptor-api localhost/cnwan/events \
 --service-account ./credentials/serv-acc.json
 ```
+
+If you want to use a configuration file, you can create the same configuration
+file as in the [example](./usage#example). For your convenience:
+
+```yaml
+adaptor:
+  host: localhost
+  port: 80
+metadataKeys:
+  - cnwan.io/traffic-profile
+serviceRegistry:
+  gcpServiceDirectory:
+    pollInterval: 10
+    region: us-west2
+    projectID: my-project
+    serviceAccountPath: ./credentials/serv-acc.json
+```
+
+**Important note**: with docker you will need to *mount* both the configuration
+file **and** the service account: therefore `serviceAccountPath` needs to be
+modified with the *mounted* path, not the one on your computer: take a look at
+its value in the example `yaml` above.
+
+Now, supposing that the configuration file is located at
+`~/Desktop/options/conf.yaml`, run
+
+```bash
+docker run \
+-v ~/Desktop/cnwan-credentials/serv-acc.json:/credentials/serv-acc.json \
+-v ~/Desktop/options/conf.yaml:/options/conf.yaml \
+my-image --conf ./options/conf.yaml
```

Comment:
This was because on docker installation there are two sections, for using the official image and one for building it yourself, so I left it as `my-image` and left the reader to change it.

Reply:
I changed it with a note to replace it for images built by the user